### DISCUSSION TITLE
Using load_page() to load multiple pages

### DISCUSSION THREAD

**Discussion by isaac-peter** [pymupdf/PyMuPDF#3565] [Looking for help]
Hello,

Are there any methods to load multiple pages concurrently? Or at minimum create a Page object with multiple pages?

For example, if ex.pdf contains 5 pages, and I want to create a Page object with pages 2 and 4, I could call some method:

doc = fitz.open('path.pdf')
two_pages = doc.method(start page, end page)

Thank you!

**Comment by JorjMcKie** [comment]
What do you want to achieve with that?
Having loaded a page won't get any valuable work done.
> Note: If you had asked to extract text from multiple page in parallel, I would have understood your intention ...

**Reply by isaac-peter** [reply]
I appreciate the prompt reply.

I have around 35,000 pdfs with results data for multiple experiments on a given day, at a given location. Within each PDF, there are multiple experiment results for a specific day at a specific location. Most experiment results are contained to one page, but sometimes run over onto 2 pages. As I'm extracting data, I would like each Page object to contain the results of one experiment, whether located on one page or two pages.

I have written code that can determine the page numbers for each experiment within a PDF. Most of the time they are one page (no problem). The problem arises when it runs over onto two pages.

**Reply by JorjMcKie** [reply]
Let me ask again:
What do you actually do with these experiment results: extract their text? Store them in a database or Excel sheet ...?
Why is it a problem if information is also contained in a second page?
It is trivial to see that a PDF has two pages (doc.page_count = 2). So in that case just also extract your stuff from the second page.

Maybe you should provide an example for each case, 1 page, 2 pages and tell what data you are extracting.

**Reply by isaac-peter** [reply]
Hello,

Thanks for the prompt reply. I figured out a solution. Thanks again.
